scripts/check-halt.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable no-console */
+const execa = require('execa')
+const got = require('got')
+const argv = require('minimist')(process.argv.slice(2))
+
+const changedPackages = require('./changed-packages')
+
+const pack = argv._[0]
+
+const containsBinary = (changes) => {
+  return !!changes.find((name) => name === 'cypress' || name.includes('@packages'))
```

Comment:
will this rebuild on changes to `scripts`? or if the root circle.yml/appveyor.yml is changed to update the binary's build process?

Reply:
Nope, currently only looks at `lerna` packages. I believe we talked about this on Friday but I forgot what we had decided on.

We can either run just the scripts tests when `scripts/...` is changed or we can run the full binary. Since they're coupled so tightly together, I think we should just run the full binary.

I'm not as convinced for the circle/appveyor files. As we import more and more packages to the monorepo, there's going to be an increasing number of changes to these files that won't be relevant to the binary.